docs/fuzz-targets.md

```diff
@@ -0,0 +1,171 @@
+# Fuzzing using fuzz targets and the CLI
+
+Creating fuzz targets and executing those via CLI commands is straight forward
+and similar to what you would expect from other fuzzers. How to do so is
+described in detail in the following sections.
+
+## Setting up Jazzer.js
+
+Before you can use Jazzer.js you have to add the required dependency
+`@jazzer.js/core` to your project.
+
+```shell
+npm install --save-dev @jazzer.js/core
+```
+
+This will install Jazzer.js and all required dependencies in your project.
```

Comment:
minor nitpick: Some clarification regarding where to execute the above command would be nice.

Reply:
done